earthly

```diff
@@ -59,7 +60,7 @@ get_latest_binary() {
     fi
 
     $frontend_bin pull --platform="$bk_platform" docker.io/earthly/buildkitd:prerelease >&2
-    $frontend_bin cp earthly_binary:"$earth_bin_path" "$bindir/earthly-prerelease" >&2
+    $frontend_bin cp earthly_"$earth_bin_path" "$bindir/earthly-prerelease" >&2
```

Comment:
is this intentional?

I just gave it a try on my machine, and it fails:

```
alex@surfperch:~/gh/earthly/earthly$ git rev-parse HEAD
06a942c51174354a1e31318166d1b9c14a4ab006

alex@surfperch:~/gh/earthly/earthly$ ./earthly +for-linux
checking for latest earthly prerelease binaries
prerelease: Pulling from earthly/earthlybinaries
9621f1afde84: Pull complete 
19c959ea7afa: Pull complete 
Digest: sha256:83bcf827df2995a685452e466c32eaa4a2b9e488c4d1e86fa5d618437676b319
Status: Downloaded newer image for earthly/earthlybinaries:prerelease
docker.io/earthly/earthlybinaries:prerelease
8fa3c939585f36dd739fb28c99fe7c40fde5b18fab9f6dc5dae8a01f0c57d647
prerelease: Pulling from earthly/buildkitd
59bf1c3509f3: Pull complete 
8d0d8be1ef72: Pull complete 
25a03d6a6f10: Pull complete 
23b93116472b: Pull complete 
a91fb26f2997: Pull complete 
05b924ea5781: Pull complete 
c884202d65a3: Pull complete 
0c4a5c06b0b1: Pull complete 
62eb7cb9cf37: Pull complete 
aa6f00fd7a83: Pull complete 
5571a0b4724c: Pull complete 
e4febe4ae775: Pull complete 
1e18899cdcd6: Pull complete 
90a220faa27a: Pull complete 
5b4e8cfbec7c: Pull complete 
097bb3a26302: Pull complete 
0b30b908808d: Pull complete 
d7b739135341: Pull complete 
4d6c569fdbd3: Pull complete 
Digest: sha256:ced85008893f7957cde535ffbc0f1fa94e6c3186b3d00e1994b978e25c3ca71e
Status: Downloaded newer image for earthly/buildkitd:prerelease
docker.io/earthly/buildkitd:prerelease
must specify at least one container source
alex@surfperch:~/gh/earthly/earthly$ echo $?
1
```

Reply:
This was not intentional.. Not sure how this change slipped in 🤔 
Great catch!

Reply:
reverted